app/src/main/java/com/helio/app/UserDataViewModel.java

```diff
@@ -109,24 +108,8 @@ public void setHubIp(String ip) {
     }
 
     public void moveCurrentMotor(int level) {
-        client.moveMotor(getCurrentMotor(), level);
-    }
-
-    public Motor getCurrentMotor() {
-        return Objects.requireNonNull(motors.getValue()).get(currentMotorId);
-    }
-
-    public void setCurrentMotor(int id) {
-        currentMotorId = id;
-    }
-
-    public void pushCurrentMotorState(Motor m) {
-        Objects.requireNonNull(motors.getValue()).put(currentMotorId, m);
-        MotorSettingsRequest motorSettingsRequest = new MotorSettingsRequest(
-                m.getName(), m.getIp(), m.isActive(), m.getBattery(), m.getLength(),
-                m.getLevel(), m.getStyle()
-        );
-        client.updateMotor(motors, currentMotorId, motorSettingsRequest);
+        // This kind of operation is deprecated and no longer available
+        throw new UnsupportedOperationException();
```

Comment:
Can we just remove this method and the "open" and "close" method instead? That seems cleaner to me, because this causes the app to crash when either of those buttons is pressed

Reply:
This is removed in the blinds settings redesign, so that was just a temporary measure.